app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,22 @@
+class ReviewsController < ApplicationController
+  def new
+  end
```

Comment:
I don't see a `new` page for reviews, which mean you shouldn't have this action! You only need an action like `new` if you're going to have a specific page that's dedicated to having a form for this thing. If the form for reviews is on a different page, you don't need a new action